### PR TITLE
LVM-377: Adds preferences module and updates tutor attributes to use it

### PR DESCRIPTION
**Pull request by nabeel-**
screen shot 2017-03-01 at 3 52 08 pm

### CODE REVIEW

**Comment by zack** [comment]
?

screen shot 2017-03-02 at 2 16 06 pm

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -40,4 +40,87 @@
   font-weight: normal;
   padding-left: 5px;
   padding-right: 10px;
+
```
Comment (zack) [inline app/assets/stylesheets/application.scss]: You need a `}` here

Comment (nabeel-) [reply app/assets/stylesheets/application.scss]: Yeah, didn't catch that when pulling in the other commits. Fixed

**Comment by zack** [comment]
lgtm 👍

**Pull request merged**